spec/p2p/reactor/reactor.md

```diff
@@ -0,0 +1,228 @@
+# Reactor API
+
+A component has to implement the [`p2p.Reactor`][reactor-interface] interface
```

Comment:
```suggestion
A component has to implement the [`p2p.Reactor` interface][reactor-interface]
```
In the original it is a bit unclear that there is a link, so the context of the following sentence is unclear.